src/components/listbox/__snapshots__/listbox.test.jsx.snap

```diff
@@ -1,12 +1,44 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`src/components/listbox/listbox has defaults 1`] = `
-<ul
```

Comment:
Is this actually supposed to be the snapshot for defaults? 😬

Reply:
yeah! check the code, it snapshots docuemnt.body